src/main/java/org/broadinstitute/hellbender/tools/ClipReads.java

```diff
@@ -303,15 +303,20 @@ public void apply( GATKRead read, ReferenceContext ref, FeatureContext featureCo
 
     @Override
     public ClippingData onTraversalSuccess(){
+        return accumulator;
+    }
+
+    @Override
+    public void closeTool() {
         if ( outputStats != null ){
             outputStats.printf(accumulator.toString());
```

Comment:
Move print statement to `onTraversalSuccess()`, but keep `outputStats.close();` here in `closeTool()`